sql/core/src/test/resources/sql-tests/inputs/typeCoercion/native/decimalArithmeticOperations.sql

```diff
@@ -83,4 +83,7 @@ select 12345678912345678912345678912.1234567 + 9999999999999999999999999999999.1
 select 123456789123456789.1234567890 * 1.123456789123456789;
 select 12345678912345.123456789123 / 0.000000012345678;
 
+-- division with negative scale operands
+select 26393499451/ 1000e6;
```

Comment:
super nit... add space `26393499451 /`

Reply:
btw, what't the result of this query in v2.3?

Reply:
sure thanks. In 2.3 this truncates the output by some digits, ie. it would return `26.393499`

Reply:
oh... thanks.